docs/source/theory/boundaryConditions.rst

```diff
@@ -943,38 +943,44 @@ field values are used for property evaluations. When flow is leaving the
 domain, the flow is advected out consistent with the choice of interior
 advection operator.
 
-.. _theory_symmetry_bc:
+.. _theory_strong_symmetry_bc:
 
-Symmetry Boundary Condition
+Strong Symmetry Boundary Condition
+++++++++++++++++++++++++++++++++++
+
+There are two implementations of the symmetry boundary condition: strong and weak. In the strong symmetry implementation, the normal velocity is set to zero at the boundary. Strong symmetry has only been implemented for a cartesian mesh, meaning it can be used for flat surfaces that are aligned with the principle Cartesian directions. It cannot be used for curved surfaces or flat surfaces that are not aligned with the principle Cartesian directions. Both the strong and weak symmetry boundary conditions have an associated error. In the strong form the associated error lies not on the boundary but in the domain.
```

Comment:
```suggestion
There are two implementations of the symmetry boundary condition: strong and weak. In the strong symmetry implementation, the normal velocity is set to zero at the boundary. Strong symmetry has only been implemented for a Cartesian mesh, meaning it can be used for flat surfaces that are aligned with the principle Cartesian directions. It cannot be used for curved surfaces or flat surfaces that are not aligned with the principle Cartesian directions. Both the strong and weak symmetry boundary conditions have an associated error. In the strong form the associated error lies not on the boundary but in the domain.
```

Reply:
Made this change here and also for next "Cartesian".